tests/xfstests/run.pm

```diff
@@ -302,7 +306,7 @@ sub run {
         select_console('root-console');
         # Save kdump data to KDUMP_DIR if not set "NO_KDUMP"
         unless (get_var('NO_KDUMP')) {
-            unless (save_kdump($test, $KDUMP_DIR, "vmcore", "kernel")) {
+            unless (save_kdump($test, $KDUMP_DIR, vmcore => 1, kernel => 1, debug => 1))) {
```

Comment:
you have two open brackets and three closing , left-over from previous version of code 
```suggestion
            unless (save_kdump($test, $KDUMP_DIR, vmcore => 1, kernel => 1, debug => 1)) {
```